### ISSUE TITLE
Loss of editing?

### ISSUE THREAD

**Issue by tingxueronghua** [VectorSpaceLab/OmniGen#188]
I implement the editing loss described in your paper and overfit the MagicBrush dataset,
However I found that the loss are very large and  the generated image quickly becomes noises. Is there any tricks required?

![image](https://github.com/user-attachments/assets/a219d78c-05be-4c6e-9ab1-1688daa030d8)

**Comment by staoxiao** [comment]
Hi, @tingxueronghua , you can refer to https://github.com/VectorSpaceLab/OmniGen/blob/main/train.py#L208-L228 for detailed settings.
